server/store.go

```diff
@@ -26,7 +26,7 @@ type ResourceKind int
 
 const (
 	leaderKind ResourceKind = iota + 1
-	storageKind
+	regionKind
```

Comment:
I think it is reasonable to use `regionKind` here, because we balance by transferring region peers anyway.